community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/Read.java

```diff
@@ -111,9 +113,9 @@ public final void nodeIndexSeek(
             {
                 switch ( q.type() )
                 {
-                case rangeGeometric:
-                case rangeNumeric:
-                    if ( !reader.hasFullValuePrecision( q ) )
+                case range:
+                    ValueGroup valueGroup = q.valueGroup();
```

Comment:
do we really need to check the value group here, isn't it enough to check the `hasFullValuePrecision`?